### PR TITLE
Cerestation Tweaks MKIII

### LINKED ISSUE #26269: Cryocells have invalid follow links (Cerestation)
`<a href=?src=[mob_779];follow=[icon-cell-off (WEST)]>(F) [Medical] cryo cell beeps, "Patient fully restored"
`

### PR DESCRIPTION
**Pull request by MMMiracles**
🆑 MMMiracles (Cerestation)
add: Departments have been given head offices for a more secure place to chill in their departments.
add: There is now a theatre on the service asteroid.
tweak: Maintenance visuals and loot have been overhauled to be more visually interesting and have more scattered bits of loot. 
fix: Various fixes with missing APCs, camera coverage, and misc things.
/🆑

- Added missing APCs to Vault and Custodial office.
- Added several cameras to areas that had sub-par camera coverage.
- Adds a sign warning the crew of the hazards of suicide hopline and more efficient methods of suicide that also increases station productivity.
- Each department has a head office for heads to stay in their department with their respective suit storage being moved there.
- Added a theatre on the service asteroid.
- Revamped maintenance visuals and loot spawns to be more visually interesting and have more junk/clutter to scavenge for whatever. Maintenance now looks more like a mixture of half-finished maintenance tunnels and dug-out areas.

Fixes #26269

### CODE REVIEW

File: _maps/map_files/Cerestation/cerestation.dmm [hunk 1]
```diff
@@ -85742,6 +85861,287 @@
 	baseturf = /turf/open/floor/plating/asteroid/airless
 	},
 /area/toxins/mixing)
+"cYd" = (
+/obj/structure/sign/biohazard{
+	desc = "A sign stating that there are better, more efficient methods of suicide that don't cause extra work for security and the janitor. Volunteer to be miner bait, be voluntary specimen for Research, or just find your nearest external airlock! ";
+	name = "SUICIDE HOPLINE ISN'T THE WAY!";
+	pixel_y = 32
+	},
```
Comment (coiax) [inline _maps/map_files/Cerestation/cerestation.dmm]: Please make this its own sign type, so other mappers can use it.

**Comment by Davidj361** [comment]
Please make it so that lawyers have access to the brig.

**Comment by MMMiracles** [comment]
@Davidj361 They should already have brig access, I addressed that issue in the first tweak PR.

**Comment by Supermichael777** [comment]
please add a security access records computer their are currently no officer accessible ones

**Comment by MMMiracles** [comment]
Gonna slap a DNM on this for a bit as I'm doing some more substantial changes while I'm at it, including the addition of a theatre on the service asteroid and some reworks on departments to include head offices, so they have somewhere moderately safe to chill inside their own department. Now would be a good time to slip in any further suggestions while I'm at it.

**Comment by Jalleo** [comment]
I suggest make the maint look more like its partially built with more girders and latticing on floors if we can lattice on asteroid tiles still. Also less lights in the probably used walking areas of maint or make them partially contructed. Thirdly that walkway between science and chapel make the left side a public garden with air but basically all the flooring is sand and only one vent and scrubbir with windoor entrance. It makes it useable and interesting difference from any other station design of the garden.

**Comment by optimumtact** [comment]
If you use [WIP] your pr gets autotagged and you can remove it to remove the tag.

**Comment by BeeSting12** [comment]
Make the changes in #26086, too, so you don't have to deal with conflicts. (Just the ones to the shuttle.)

**Comment by MMMiracles** [comment]
Ready for review/merging.

**Comment by GunHog** [comment]
Recommended changes/Mapping notes:
- Put the cameras on either side of the central pillar in the Aux Base. Do not put the camera on the edge of the base's floor, because players need to build there, which will cause them to obstruct their own camera view.
- There is a camera missing on the bottom left of the AI chamber. You do not notice it is missing because of the AI's internal camera.
- **Where is the Head of Security's office?**
- The RD's office is rather dark. Did you forgot to add a light?
- The Science to Arrivals bridge (both ends) is missing a few blocks of camera coverage.
- The southern end of the Arrivals to Medical bridge is missing some coverage.
- The Engineering to Medical bridge is missing coverage on both ends.
- There are two CE hardsuits, one in the CE's office and one in the engine room.
- The entrance to the Engineering SMES room has a blind spot at the airlock.
- The gas containment rooms in Atmos are not visible by camera. Intended?
- CMO's office is missing a camera.

**Comment by MMMiracles** [comment]
HoS doesn't get an office as hes probably the only head who can chill in their department with little fear of being jumped by an underling, while having the advantage of being decently armed and multiple people he can trust to have his back (loyalty implants).

**Comment by Jordie0608** [comment]
Any more to add to this or good to go? Also have you been using the map-merger? The diff is a bit suspect.

**Comment by MMMiracles** [comment]
I have been using the merger, but I shifted around a lot of stuff and redid a lot of maintenance so the diff is going to be high.

Everything is good to go for merging.

**Comment by GunHog** [comment]
Was not including cameras for the atmos air tanks a design choice? I find it difficult to set up a plasma flood or something as AI due to it being harder to see what I am doing.

**Pull request merged**